routing_common/vehicle_model.hpp

```diff
@@ -55,8 +56,9 @@ enum class HighwayType : uint16_t
   RouteFerry = 259,
   HighwayTertiaryLink = 272,
   HighwayBusway = 858,    // reserve type here, but this type is not used for any routing by default
-  RailwayRailMotorVehicle = 994,
+  RailwayRailMotorVehicle = 994, // TODO: to be deprecated, see mapcss-mapping.csv.
   RouteShuttleTrain = 1054,
+  RailwayMotorVehicle = 1501,
```

Comment:
Do not add new entry, just change value of the existing one. To avoid bloat changes.
Also don't rename.